Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Commands.cpp

```diff
@@ -21,11 +21,23 @@
 
 int CLuaFunctionDefs::AddCommandHandler ( lua_State* luaVM )
 {
-//  bool addCommandHandler ( string commandName, function handlerFunction, [bool caseSensitive = true] )
-    SString strKey; CLuaFunctionRef iLuaFunction; bool bCaseSensitive;
-
+//  bool addCommandHandler ( string/table commandName, function handlerFunction, [bool caseSensitive = true] )
+    SString strKey;
+    std::vector < SString > vecKey;
+    bool bType;
```

Comment:
Please rename the variable to convey the proper meaning of this flag. For example: **bUseKeyVector**, give it a default value and flip the false/true states below.